packages/shared/src/components/sidebar/sections/BookmarkSection.tsx

```diff
@@ -39,7 +39,12 @@ export const BookmarkSection = ({
       ) => {
         event.preventDefault();
 
-        openModal({ type: LazyModal.BookmarkFolderSoon, props: {} });
+        openModal({
+          type: LazyModal.BookmarkFolder,
+          props: {
+            onSubmit: () => {},
```

Comment:
Let's just add a `TODO:` comment to make it easier to track.

Reply:
Handler has already been added in a WIP branch :)